x/harvest/keeper/liquidation_test.go

```diff
@@ -0,0 +1,296 @@
+package keeper_test
+
+import (
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/crypto"
+	tmtime "github.com/tendermint/tendermint/types/time"
+
+	"github.com/kava-labs/kava/app"
+	"github.com/kava-labs/kava/x/harvest"
+	"github.com/kava-labs/kava/x/harvest/types"
+	"github.com/kava-labs/kava/x/pricefeed"
+)
+
+// type LiquidationTestSuite struct {
+// 	suite.Suite
+// }
+
+func (suite *KeeperTestSuite) TestKeeperLiquidation() {
+	type args struct {
```

Comment:
additional verification args:

```
expectedKeeperCoins // additional coins keeper address should have after successfully liquidating position
expectedAuctions // the auctions we should expect to find have been started
expectedBorrowerCoins // additional coins (if any) the borrower address should have after successfully liquidating position
```

Reply:
Question about testing approaches: currently the `expectedKeeperCoins` value is being calculated during the test as `Borrow coins * keeper reward % = expectedKeeperCoins` (mimicking the on-chain logic). If the keeper's current module balance equals `initialKeeperCoins + expectedKeeperCoins` then the test passes. In general, is it better to dynamically mimic the on-chain logic for flexible tests or to manually preset expected values so that we're more likely to catch any issues?

Reply:
Added with `expectedAuctions` and checked auction state in https://github.com/Kava-Labs/kava/pull/731/commits/530238cc8bc0d072e7ffbb25fd5445f96b5d172c.

I preset the `expectedKeeperCoins` in https://github.com/Kava-Labs/kava/pull/731/commits/7d6e3021e962605003db96ffe8b008c025570d69. If we want to go with the dynamic approach should be easy to revert the single commit.